### PR TITLE
Update databinding gradle config

### PR DESCRIPTION
**Pull request by diegoaces**
Fix DSL element 'android.dataBinding.enabled' is obsolete and has been replaced with 'android.buildFeatures.dataBinding' warning log

### CODE REVIEW

**Comment by diegoaces** [comment]
@googlebot I signed it!

File: app/build.gradle [hunk 1]
```diff
@@ -22,8 +22,10 @@ apply plugin: 'androidx.navigation.safeargs.kotlin'
 
 android {
     compileSdkVersion rootProject.compileSdkVersion
-    dataBinding {
-        enabled = true
+    buildFeatures {
+        dataBinding = true
+        // for view binding :
```
Comment (tiembo) [inline app/build.gradle:27]: Can you remove the commented out code?

**Comment by tiembo** [comment]
Thank you for the PR!  Added some comments.

I'll look into the CLA warning.

**Comment by diegoaces** [comment]
@tiembo  comments removed

**Comment by tiembo** [comment]
The PR looks OK, though the CLA does not pass.  Can the 'Dacuna' user also sign the CLA?

**Comment by tiembo** [comment]
Addressed in #622

**Pull request closed without merging**